### PR TITLE
Update 2021 curriculum links to be version agnostic where possible

### PR DESCRIPTION
**Pull request by davidsbailey**
Starts [PLAT-1861](https://codedotorg.atlassian.net/browse/PLAT-1861). Updates a few links to 2021 curriculum to be version agnostic, and makes some incidental fixes along the way. See individual commit messages for more details.

## Testing story

* manually verified that links like https://studio.code.org/s/aiml?viewAs=Instructor still work and preserve the url param

## Deployment strategy

safe to deploy today

## Follow-up work

Several more PRs forthcoming.

### CODE REVIEW

File: dashboard/config/scripts/levels/VPL - CSD Module 5 - Differentiation_2022.level [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
...n\r\n*“Creating learning environments that are accessible and welcoming of students’ identities, backgrounds, differences and perspectives without barriers or judgment.  This means actively attending to gender, race,  ethnicity, ability or socioeconomic status.”*\r\n\r\n## Goals\r\n\r\n* Identify teacher support available within lessons and the curriculum that support differentiation to promote inclusion\r\n* Identify current classroom strategies you use that meet various needs of learners and promote inclusion\r\n\r\n## Tasks\r\n\r\n* Review the lesson plan for the model lesson, <a href=\"https://studio.code.org/s/csd5-2021/lessons/5\" target=\"_blank\">Lesson 5:  Representing Numbers</a>\r\n* Preview the <a href=\"https://docs.google.com/document/d/15Y6Mo3fMUtHq16SSskABgcYE-5xfk-Ewv0z_o4uaLdI/edit\" target=\"_blank\">CSD Guide to Differentiation 2021</a>\r\n\r\n## Reflection\r\n\r\n**Prompts:**\r\n* What teacher supports and strategies are available within Lesson 5 (e.g. Teacher Tips, etc) and the CSD Guide to Differentiation 2021?\r\n* What strategies do you currently use in your classroom that promote inclusion and meet the unique learning needs of your students? \r\n\r\n\r\n",
```
Comment (dmcavoy) [inline dashboard/config/scripts/levels/VPL - CSD Module 5 - Differentiation_2022.level]: I'm not sure I can tell what changed here. Can you help me?

**Review by dmcavoy — APPROVED** [review]
Looks good besides the one file where I could not tell what changed.

File: dashboard/config/scripts/levels/VPL - CSD Module 5 - Differentiation_2022.level (hunk 1 above)
Comment (davidsbailey) [reply dashboard/config/scripts/levels/VPL - CSD Module 5 - Differentiation_2022.level]: apologies, this was just http --> https:
Screen Shot 2022-06-28 at 6 41 02 PM

Comment (dmcavoy) [reply dashboard/config/scripts/levels/VPL - CSD Module 5 - Differentiation_2022.level]: Oh ok!

**Comment by davidsbailey** [comment]
oops, I accidentally merged the other PR into this one before merging this one, so this PR now looks like there is a lot more in it than when you reviewed it. shouldn't be a problem so just FYI. :-)

**Pull request merged**